dask_planner/src/sql/logical/union.rs

```diff
@@ -0,0 +1,32 @@
+use datafusion::logical_expr::logical_plan::Union;
+pub use datafusion::logical_expr::LogicalPlan;
+
+use pyo3::prelude::*;
+
+#[pyclass(name = "Union", module = "dask_planner", subclass)]
+#[derive(Clone)]
+pub struct PyUnion {
+    union: Union,
+}
+
+#[pymethods]
+impl PyUnion {
+    #[pyo3(name = "all")]
+    pub fn all(&mut self) -> PyResult<bool> {
+        println!("{:?}", self.union.inputs[0]);
+        println!("{:?}", self.union.schema.metadata());
+        println!("{:?}", self.union.alias);
```

Comment:
`println!` statements should be removed. If you want you could implement `__str__` impl like 

```
fn __repr__(&self) -> &'static str {
        format!("{:?}\n{:?}\n{:?}", 
                        self.union.inputs[0],
                        self.union.schema.metadata(),
                        self.union.alias)
}
```

Reply:
Thanks Jeremy, disabled this method for future use along with a TODO.